azerothcore/azerothcore-wotlk
# fix(DB/SAI): Missings spells for few creatures in Blade's Edge Mountains

## Changes Proposed:

This PR proposes changes to:

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/19594 and https://github.com/chromiecraft/chromiecraft/issues/5408#issue-1666880763

## SOURCE:

The changes have been validated through:

## Tests Performed:

This PR has been:


## How to Test the Changes:



1. .go creature 19993 and check that the creature cast knockdown
2. .go creature 22160 and check that the creature cast knockdown
3. .go creature 19995 and check that the creature cast Cleave

## Known Issues and TODO List:


- [X] Comparing to the "confirmed" issue, I wasn't able to get : Fire Nova Totem with Bladespire Shaman / Bloodmaul Buzz with Bloodmaul Mauler on live servers (retail and cata) --> Edit: WoWhead sources so I think that my check on live servers can confirm that these creatures doesn't cast mentioned spells.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
